decompose/src/commonMain/kotlin/com/arkivanov/decompose/router/RouterExt.kt

```diff
@@ -37,3 +37,6 @@ fun <C : Any> Router<C, *>.bringToFront(configuration: C) {
         stack.filterNot { it::class == configuration::class } + configuration
     }
 }
+
+val <C : Any> RouterState<C, *>.activeConfiguration: C get() = activeChild.configuration
+val <C : Any> Router<C, *>.activeConfiguration: C get() = state.value.activeConfiguration
```

Comment:
It looks too specific, let's add just one property `val <C : Any, T : Any> Router<C, T>.activeChild: Child.Created<C, T>`. It should ease accessing the active child, and dedicated extensions can be added manually on the client-side.

Reply:
Thanks, implemented those changes